### DISCUSSION TITLE
Files are food names

### DISCUSSION THREAD

**Discussion by hnaboulsi** [benawad/dogehouse#1617] [General]
Why are the file's names related to arabic food, who came up with the idea? It's a bit funny lol

**Comment by ajhalili2006** [comment]
> I don't know if this answer is correct, but if it's Stack Overflow please improve this answer for the future.

Mostly, @benawad name these directories as Arabic foods because either he's born somewhere  in Middle East or his family lived there in the past.

**Reply by ajhalili2006** [reply]
See also: https://github.com/benawad/dogehouse/discussions/249
